portfolio/src/main/webapp/index.html

```diff
@@ -5,6 +5,7 @@
     <title>Capstone Project</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js" defer></script>
+    <script src="waypoints.js" defer></script>
```

Comment:
this should go above script.js since the methods in there will call the ones in waypoints.js

Reply:
Noted, thanks!